### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ljfreelancer88**
Added basic systemd configurations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,21 @@ catch(\Exception $e) {
...
 ```
 
+#### Systemd configuration for Consumer / Worker
+Note that this does not aim to cover all possible scenarios or configurations.
+```cli
+[Unit]
+Description=My App Worker
+
+[Service]
+User=root
```
[the remark this answers is not in the document]
Comment (ljfreelancer88) [reply README.md:85]: Ow! I've overlooked that. Feel free to change it.

[the remark this answers is not in the document]
Comment (SamMousa) [reply README.md:85]: I don't use this setup, I prefer to run things in containers. I propose you change the username to something appropriate!

[the remark this answers is not in the document]
Comment (ljfreelancer88) [reply README.md]: I will. Thank you 🙏 Maybe, you can your approach too if that's OK

**Review by ljfreelancer88 — COMMENTED** [review]
Changed user

**Comment by SamMousa** [comment]
Thanks, sorry for letting this cook for a bit.

**Pull request merged**